sdk/resourcemanager/azure-resourcemanager/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
why is this file under test/resources folder rather than sample/resources folder?

Reply:
this is for enabling feature on mockito to enable mocking of final class.

see https://github.com/Azure/azure-sdk-for-java/wiki/Unit-Testing#mocking-final-classes and its reference to mockito.